content/en/docs/tasks/administer-cluster/kubelet-config-file.md

```diff
@@ -53,6 +53,10 @@ For a trick to generate a configuration file from a live node, see
 
 ## Start a Kubelet process configured via the config file
 
+{{< note >}}
+If you use kubeadm to initialize your cluster, use the kubelet-config while creating your cluster with `kubeadmin init`. See https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/kubelet-integration/ for details.
```

Comment:
```suggestion
If you use kubeadm to initialize your cluster, use the kubelet-config while creating your cluster with `kubeadmin init`.
See [configuring kubelet using kubeadm](/docs/setup/production-environment/tools/kubeadm/kubelet-integration/) for details.
```

Reply:
ok for me. Is there a way to squash this, so I do not have two commits for this change?

Reply:
/label tide/merge-method-squash